InscryptionAPI/Triggers/CustomTriggerPatches.cs

```diff
@@ -6,209 +6,281 @@
 using System.Text;
 using HarmonyLib;
 using UnityEngine;
+using System.Security.Cryptography.X509Certificates;
 
 namespace InscryptionAPI.Triggers
 {
     [HarmonyPatch]
-    internal static class CustomTriggerPatches
+    public static class CustomTriggerPatches
```

Comment:
It seems like you've just copied this class in its entirety from your own branch, since this reverts things like making all the patches private

Reply:
yeah i did sorry about that. will fix